src/components/Dropdown/Dropdown.tsx

```diff
@@ -197,36 +215,35 @@ export const DropdownPanel = <T extends { id: string }>({
                         />
                     </>
                 ))}
-                {nullable(items.length && title, () => (
-                    <Text size="s" weight="bold" className={s.DropdownSuggestionsTitle}>
-                        {title}
-                    </Text>
-                ))}
 
-                <ListView onKeyboardClick={handleChange}>
-                    {items.map((item) => (
-                        <ListViewItem
-                            key={item.id}
-                            value={item}
-                            renderItem={(props) => (
-                                <MenuItem
-                                    onClick={() => {
-                                        handleChange(item);
-                                        onClose();
-                                    }}
-                                    {...props}
-                                    hovered={props.active}
-                                    iconLeft={nullable(selectable, () => (
-                                        <Dot color={value?.id === item.id ? 'var(--check-checked)' : 'transparent'} />
-                                    ))}
-                                    {...comboboxItem.attr}
-                                >
-                                    {renderItem?.({ ...props, item })}
-                                </MenuItem>
-                            )}
-                        />
-                    ))}
-                </ListView>
+                <AutoComplete
+                    items={items}
+                    mode={props.mode}
+                    onChange={handleChange}
+                    value={value}
+                    renderItem={({ item, isSelected, active, onChange, ...props }) => (
+                        <MenuItem
+                            onClick={() => {
+                                onChange();
+                                if (multiple) return;
+                                onClose?.();
+                            }}
+                            {...props}
+                            multiple={multiple}
+                            hovered={active}
+                            selected={valueMap[item.id]}
+                            selectable={selectable}
+                            iconLeft={nullable(selectable, () => {
+                                return iconLeft || (multiple && <Checkbox id={item.id} checked={isSelected} />);
```

Comment:
Use `defaultChecked` instead of `checked` if `<Checkbox />` don't  getting `onClick` fn